endpointmanager/pkg/endpointmanager/postgresql/store_test.go

```diff
@@ -38,9 +38,9 @@ func setupConfig() {
 
 	viper.SetDefault("dbhost", "localhost")
 	viper.SetDefault("dbport", 5432)
-	viper.SetDefault("dbuser", "postgres")
-	viper.SetDefault("dbpass", "postgrespassword")
-	viper.SetDefault("dbname", "postgres")
+	viper.SetDefault("dbuser", "lantern")
+	viper.SetDefault("dbpass", "lanternpassword")
+	viper.SetDefault("dbname", "lantern")
```

Comment:
this is for a different part of code.

in `postgresql/store.go`, add a corresponding line to the following:

```golang

// Ensure Store implements endpointmanager.FHIREndpointStore.
var _ endpointmanager.FHIREndpointStore = &Store{}

// Ensure Store implements endpointmanager.HealthITProductStore.
var _ endpointmanager.HealthITProductStore = &Store{}

// Ensure Store implements endpointmanager.ProviderOrganizationStore.
var _ endpointmanager.ProviderOrganizationStore = &Store{}
```